### DISCUSSION TITLE
Sync?

### DISCUSSION THREAD

**Discussion by ithakaa** [athensresearch/athens#1377] [Bug Reports]
Given the current covid landscape, working from home is a thing going forward, not because it's necessary but because it's available.

How do we sync notes between workstation?

**Comment by joelhans** [comment]
I personally use OneDrive to sync my notes, and I believe others are using Dropbox. You can just move your `index.transit` file into that folder and open it from the new location.

I've had pretty good success with it so far, as long as I don't do too much editing of the database in quick succession from multiple systems. Athens warns you when you have a page/block open and your sync tool of choice downloads an update to `index.transit` that will change said page/block.

We have some (sparse) documentation about syncing here: https://athensresearch.gitbook.io/handbook/community/athens-guide/getting-started/installation-guide#hosting-your-own-data

Let me know about your experiences with syncing so I can make that section better!

**Reply by ithakaa** [reply]
I would rather use a backend server as you suggested but the documentation is clear that it's broken.

Will try Dropbox

Thanks

**Reply by joelhans** [reply]
If you're interested in the backend, you'll be curious about this project board that will integrate self-hosted real-time collaboration into this repo and make deploying it a lot easier: https://github.com/athensresearch/athens/projects/16

The good news for you is that if you start now with Dropbox, the migration process will just be copying the database over to the backend server and telling Athens to load it from your server rather than Dropbox!
